app/views/admin/merchants/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Admin: Merchants</h1>
```

Comment:
I agree that we should continue to label each view with a header describing what the page is.  Just so we don't get lost when navigating pages.